.github/workflows/testsuite.yaml

```diff
@@ -3,35 +3,6 @@ on: [push, pull_request]
 
 
 jobs:
-  linter:
-    name: Linter
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        # Adding -l {0} helps ensure conda can be found properly.
-        shell: bash -l {0}
-    env:
-      PYTHON: 3.8
-      ENV_NAME: linting
-    steps:
-      - uses: actions/checkout@master
-        with:
-          fetch-depth: 1
-
-      - name: Setup Miniconda
-        uses: conda-incubator/setup-miniconda@v2.0.0
-        with:
-          auto-update-conda: true
-          miniconda-version: "latest"
-          python-version: ${{ env.PYTHON }}
-          environment-file: ci/${{ env.ENV_NAME }}.yaml
-          activate-environment: ${{ env.ENV_NAME }}
-
-      - name: Linting
-        run: |
-          pre-commit install
-          pre-commit run -a
-
```

Comment:
Are you removing this because the assumption is anyone pushing is using pre-commit therefore it should always pass or something else? I feel like it is still good to run the pre-commit checks because you can recommend to developers to install pre-commit but you can't _make_ them right?

Reply:
I removed it because I turned on pre-commit.ci which runs this check for us much faster and with no configuration :-)

Reply:
oh yeah duh. I even _looked_ at it.